FrontEnd/src/hooks/useAuth.js

```diff
@@ -1,6 +1,108 @@
 import { useContext } from 'react';
```

Comment:
Seems that it is not a react hook anymore. Its rather context provider. Could we use appropriate file name than

Reply:
Do we really have to rename this file? I think we have here both context provider (AuthProvider) and custom hook (useAuth) to provide access to context. Can I leave it unchanged?

Reply:
`useAuth` shows us that this file is the hook. But in fact, it is a Context Provider. I would recommend to rename it. There are 3 lines of the `useAuth` hook where we use this context.

Reply:
To do not mix white and black we may extract context provider to separate file and leave this hook unchanged

Reply:
Done